### PR TITLE
Display complex xrefs on scrollable tables

### PR DESCRIPTION
**Pull request by jmedinaebi**
Replace the 'Show all' buttons with scrollable tables. It is deployed ion github, go to the pages below to check the changes.

I have also replace `@Input` with `input<...>()` in the files I've modified.

- ChEMBL xrefs (we don't have any complex with enough of these xrefs to be scrollabe)
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-2158
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-2007

- Disease xrefs
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-383
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-2158 (5 xrefs, not scrollable)

- GO cellular component xrefs
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-329
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-2158 (1 xref, not scrollable)

- Agonists and antagonists xrefs
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-362 (not scrollable)

- Catalytic activity xrefs
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-2469
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-2007 (not scrollable)

- GO molecular function and biological process xrefs
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-296
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-362
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-383 (not scrollable)

- Ligands xrefs
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-3137
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-296 (not scrollable)

- EMDB and PDB xrefs
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-5682 (over 200 xrefs of each)
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-383 (not scrollable)
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-329 (single PDB xref, not scrollable)

- Synonyms
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-2158
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-329 (not scrollable)

- Complex components
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-383 (4 components, not scrollable)
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-329 (11 components, scrollable)
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-5602 (with subcomplexes, not scrollable)
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-6164 (with subcomplexes, scrollable when all expanded) 

- Reactome xrefs
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-2158
  - https://complex-portal.github.io/complex-portal-view/complex/CPX-2469 (not scrollable)

### CODE REVIEW

File: src/app/complex/complex-details/complex-properties/pdb-crossreferences/pdb-crossreferences.component.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+tbody {
+  height: 350px;
```
Comment (EliotRagueneau) [inline src/app/complex/complex-details/complex-properties/pdb-crossreferences/pdb-crossreferences.component.scss:2]: A bit unsure of those constant px size, is it working well in all viewing contexts?

**Review by EliotRagueneau — APPROVED** [review]
Looks great, maybe the only downside is if we have many references, it could impact performance because we have too much nodes, but I don't think we would ever have so many items that it would happen. We could think of a "intelligent" component who's able to display the first elements and only display the elements afterwards if the user actually scroll in them, but it's probably overkill for now as this problem will probably not occur any time soon.

**Comment by EliotRagueneau** [comment]
Just saw that Angular Material has native support for that, using the cdk-scrolling : https://material.angular.io/cdk/scrolling/overview

If we ever see performance issue because we have too many items in our different list, we could try to use this strategy. Not requirred right now I think, but good to know it exists

File: src/app/complex/complex-details/complex-properties/pdb-crossreferences/pdb-crossreferences.component.scss (hunk 1 above)
Comment (jmedinaebi) [reply src/app/complex/complex-details/complex-properties/pdb-crossreferences/pdb-crossreferences.component.scss:2]: I don't know where it is exactly coming from, but the PDB xrefs table already had a min size, so even if we only had 1 xref, the table will be the full height to match the pdb widget on the left side. From what I've seen, this is working fine.

**Pull request merged**